cmd/root.go

```diff
@@ -55,6 +55,15 @@ func init() {
 		log.Fatal(err)
 	}
 
+	// Working directory
+	rootCmd.PersistentFlags().StringP("dir", "C", "./", "Working directory")
```

Comment:
Shall we change this to `context` as we already have something as `dir`

Reply:
Sounds good.